src/plugins/git/loadRepository.test.js

```diff
@@ -0,0 +1,156 @@
+// @flow
+
+import fs from "fs";
+import mkdirp from "mkdirp";
+import path from "path";
+import tmp from "tmp";
+
+import type {GitDriver} from "./loadRepository";
+import {localGit, loadRepository} from "./loadRepository";
+
+const cleanups: (() => void)[] = [];
+afterAll(() => {
+  cleanups.forEach((f) => {
```

Comment:
(nit): why not `cleanups.forEach((f) => f())`?

Reply:
That lambda is equivalent to `(f) => { return f(); }`, which is not what I mean. In this case, the effect is the same (because JavaScript), but I prefer to follow Google style and only use the expression form when you actually want an expression.